x-pack/plugins/cases/public/components/user_actions/markdown_form.test.tsx

```diff
@@ -33,156 +32,37 @@ const defaultProps = {
   });
 
-  it('Save button click calls onSaveContent and onChangeEditable when text area value changed', async () => {
-    const wrapper = mount(
-      <TestProviders>
-        <UserActionMarkdown {...defaultProps} />
-      </TestProviders>
-    );
+  it('Renders markdown correctly when in edit mode', async () => {
+    appMockRenderer.render(<UserActionMarkdown {...{ ...defaultProps, isEditable: true }} />);
 
-    wrapper
-      .find(`.euiMarkdownEditorTextArea`)
-      .first()
-      .simulate('change', {
-        target: { value: newValue },
-      });
-
-    wrapper.find(`button[data-test-subj="user-action-save-markdown"]`).first().simulate('click');
-
-    await waitFor(() => {
-      expect(onSaveContent).toHaveBeenCalledWith(newValue);
-      expect(onChangeEditable).toHaveBeenCalledWith(defaultProps.id);
-    });
-  });
-
-  it('Does not call onSaveContent if no change from current text', async () => {
-    const wrapper = mount(
-      <TestProviders>
-        <UserActionMarkdown {...defaultProps} />
-      </TestProviders>
-    );
-
-    wrapper.find(`button[data-test-subj="user-action-save-markdown"]`).first().simulate('click');
-
-    await waitFor(() => {
-      expect(onChangeEditable).toHaveBeenCalledWith(defaultProps.id);
-    });
-    expect(onSaveContent).not.toHaveBeenCalled();
-  });
-
-  it('Save button disabled if current text is empty', async () => {
-    const wrapper = mount(
-      <TestProviders>
-        <UserActionMarkdown {...defaultProps} />
-      </TestProviders>
-    );
-
-    wrapper
-      .find(`.euiMarkdownEditorTextArea`)
-      .first()
-      .simulate('change', {
-        target: { value: emptyValue },
-      });
-
-    await waitFor(() => {
-      expect(
-        wrapper.find(`button[data-test-subj="user-action-save-markdown"]`).first().prop('disabled')
-      ).toBeTruthy();
-    });
-  });
-
-  it('Cancel button click calls only onChangeEditable', async () => {
-    const wrapper = mount(
-      <TestProviders>
-        <UserActionMarkdown {...defaultProps} />
-      </TestProviders>
-    );
-    wrapper.find(`[data-test-subj="user-action-cancel-markdown"]`).first().simulate('click');
-
-    await waitFor(() => {
-      expect(onSaveContent).not.toHaveBeenCalled();
-      expect(onChangeEditable).toHaveBeenCalledWith(defaultProps.id);
-    });
+    expect(screen.getByTestId('editable-save-markdown')).toBeInTheDocument();
+    expect(screen.getByTestId('editable-cancel-markdown')).toBeInTheDocument();
   });
 
   describe('useForm stale state bug', () => {
-    let appMockRenderer: AppMockRenderer;
     const oldContent = defaultProps.content;
     const appendContent = ' appended content';
     const newContent = defaultProps.content + appendContent;
 
-    beforeEach(() => {
-      appMockRenderer = createAppMockRenderer();
-    });
-
-    it('creates a stale state if a key is not passed to the component', async () => {
```

Comment:
This issue of stale state doesn't happen anymore as `fieldValue` is passed to `editable-markdown-renderer`, which checks if the` filedValue.content `is different than the `content` props, then it updates the value.